user-docs/latest/users-roles-permissions/managing-administrators.md

```diff
@@ -0,0 +1,99 @@
+- or access information regarding an administrator account, and edit it (see [Editing an account](#editing-an-account)).
+
+For each administrator listed in the table, their main account information are displayed, including name, email and attributed role. The status of their account is also indicated: active or inactive, depending on whether the administrator has already logged in to activate the account or not.
+
+<!---
+### Filtering accounts
+
+Right above the table, on the left side of the interface, a **Filters** button is displayed. It allows to set one or more condition-based filters, which add to one another (i.e. if you set several conditions, only the administrator accounts that match all the conditions will be displayed).
+
+To set a new filter:
+
+1. Click on the **Filters** button.
+2. Click on the 1st drop-down list to choose the field on which the condition will be applied.
+3. Click on the 2nd drop-down list to choose the type of condition to apply.
+4. Enter the value of the condition in the 3rd box.
+5. Click on the **Add filter** button.
+
+::: tip NOTE
+When active, filters are displayed next to the **Filters** button. They can be removed by clicking on the delete icon.
+:::
+--->
+
+## Creating a new account
+
+On the top right side of the *Administration panel > Users* interface, a **Create new User** button is displayed. It allows to create a new administrator account for your Strapi application.
+
+To create a new administrator account:
+
+1. Click on the **Create new User** button.
+2. In the *Add new user* window, fill in the following information about the new administrator:
+
+| User information  | Instructions                                                                                            |
+| ----------------- | ------------------------------------------------------------------------------------------------------- |
+| First name        | (mandatory) Write the administrator's first name in the textbox.                                        |
+| Last name         | (mandatory) Write the administrator's last name in the textbox.                                         |
+| Email             | (mandatory) Write the administrator's complete email address in the textbox.                            |
+| User's role       | (mandatory) Choose from the drop-down list the role to attribute to the new administrator.              |
+
+3. Click on the **Create user** button in the bottom right corner of the *Add new user* window.
+4. A URL appears at the top of the window: it is the URL to send the new administrator for them to log in for the first time to your Strapi application. Click the copy button ![icon copy](../assets/users-permissions/icon_duplicate.png) to copy the URL.
+5. Click on the **Continue** button in the bottom right corner to finish the new administrator account creation. The new administrator should now be listed in the table.
+
```

Comment:
I would add a tip saying that you can access the URL at any time by clicking on the Edit of an account until the account is activated.

To make sure that users are not lost if they fail to copy it the first time.